src/OmniSharp/Api/v1/Highlighting/OmnisharpController.Highlighting.cs

```diff
@@ -48,7 +48,7 @@ public partial class OmnisharpController
                 }
                 else
                 {
-                    foreach (var line in request.Lines)
+                    foreach (var line in request.Lines.Where(z => z <= text.Lines.Count))
```

Comment:
It's just a guard if the editor sends too many lines.  This was was something I saw in atom when copying and pasting an entire files contents.